authui/src/authflowv2/base.css

```diff
@@ -76,6 +76,12 @@
     --typography-label-medium__line-height: 1.25em;
     --typography-label-medium__letter-spacing: 0.03125em;
     --typography-label-medium__font-weight: 600;
+
+    --typography-body-small__font-family: inherit;
+    --typography-body-small__font-size: 0.75rem;
+    --typography-body-small__line-height: 1rem;
+    --typography-body-small__letter-spacing: 0.025rem;
```

Comment:
Use `em` as unit because it should be related to the font size of the current element not the root element.